resources/views/welcome.blade.php

```diff
@@ -22,7 +22,7 @@
             </ul>
 
             <section class="bg-white rounded-b-lg">
-                @foreach ($behind as $project)
+                @foreach ($projects as $project)
                     <ul class="flex list-reset p-4 border-t border-smoke bg-red-100">
```

Comment:
`bg-red-100` is no longer valid here since current projects still have a red background.

Reply:
Removed: https://github.com/tighten/checkmate/pull/31/commits/1173ec5ec9d4c12f4897d83fac1441b6df94ac0d